doc/doc-site/docs/off-chain/python-sdk.md

```diff
@@ -1,3 +1,97 @@
 # Python SDK
 
-More docs coming soon!
+The code for the Python SDK lives in `/econia/src/python/sdk/econia_sdk` ([link](https://github.com/econia-labs/econia/tree/main/src/python/sdk/econia_sdk)). There are 2 primary packages (`econia_sdk.entry` and `econia_sdk.view`) complemented by two secondary imports
+
+# Primary Packages
+## `econia_sdk.entry`
+
+This package contains helpers for the `public entry` (that is, "targetable by transactions") functions possessed by the Econia exchange. Each method's name corresponds to the name of a public entry function in one of the following Econia Move modules:
+
+- `incentives` ([move](https://github.com/econia-labs/econia/blob/main/src/move/econia/sources/incentives.move)) ([python](https://github.com/econia-labs/econia/blob/main/src/python/sdk/econia_sdk/entry/incentives.py))
```

Comment:
Can you put these in a table?

See the markdown source for the events doc page for the syntax

(Note that the formatter will auto-justify the table)